src/index.tsx

```diff
@@ -7,6 +7,7 @@ import web3Onboard from "shared/utils/web3Onboard"
 // no-extraneous-dependencies.
 // eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Why we import posthog-js module as dev dependency and we use every time: ` eslint-disable-next-line import/no-extraneous-dependencies` ?